packages/portalnetwork/test/networks/history/spec-test-runner.spec.ts

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-console */
+import { bytesToHex, hexToBytes } from '@ethereumjs/util'
+import { readFileSync, readdirSync, statSync } from 'fs'
+import yaml from 'js-yaml'
+import { join, resolve } from 'path'
+import { afterAll, beforeAll, describe, it } from 'vitest'
+import type { HistoryNetwork } from '../../../src/index.js'
+import {
+  HistoryNetworkContentType,
+  PortalNetwork,
+  decodeHistoryNetworkContentKey,
+  getContentKey,
+} from '../../../src/index.js'
+
+describe.skip('should run all spec tests', () => {
```

Comment:
Maybe insignificant, but I believe that even when tests have `.skip` vitest does the pre-run processing and test set up.  It might be better to instead add this to the `exclude` list in the vitest config flie.